### PR TITLE
Improve PEGELONLINE docs according to new quality scale

### PR DESCRIPTION
**Pull request by mib1185**
## Proposed change

SSIA

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

## Summary by CodeRabbit

- **New Features**
	- Added a "Data fetching and limitations" section detailing polling frequency and usage guidelines.
	- Introduced new configuration steps for location and station selection with required parameters.
	- Added a "Usage example" section for YAML configuration related to flood alerts.

- **Documentation**
	- Enhanced clarity in the supported sensors section based on measurement station capabilities.
	- Expanded "Troubleshooting" section with instructions for enabling debug logging and diagnostics.

### CODE REVIEW

File: source/_integrations/pegel_online.markdown [hunk 1]
```diff
@@ -34,3 +62,31 @@ Currently, the integration supports the following sensors:
...
+  - trigger: numeric_state
+    entity_id:
+      - sensor.dresden_elbe_water_level
+    above: 500
+actions:
+  - action: notify.persistent_notification
+    metadata: {}
+    data:
+      message: Flood alert level 2 reached!
+```
```
[the remark this answers is not in the document]
Comment (mib1185) [reply source/_integrations/pegel_online.markdown:80]: the example should be held simple

File: source/_integrations/pegel_online.markdown [hunk 2]
```diff
@@ -18,11 +18,39 @@ ha_integration_type: service
...
+{% configuration_basic %}
+Latitude:
+    description: "The latitude of the center of the search area (_automatically filled by the location picker_)."
+    required: true
+Longitude:
+    description: "The longitude of the center of the search area (_automatically filled by the location picker_)."
+    required: true
+Radius:
+    description: "Search radius (_in km_)"
+    required: true
```
[the remark this answers is not in the document]
Comment (mib1185) [reply source/_integrations/pegel_online.markdown]: This is ensured by the implementation, which limits the range

File: source/_integrations/pegel_online.markdown [hunk 3]
```diff
@@ -18,11 +18,39 @@ ha_integration_type: service
...
 {% include integrations/config_flow.md %}
 
+### Step 1 - Location selection
+
+Select the area, where you want to search for available water measuring stations
+
+{% configuration_basic %}
+Latitude:
+    description: "The latitude of the center of the search area (_automatically filled by the location picker_)."
+    required: true
```
Comment (frenck) [inline source/_integrations/pegel_online.markdown]:
```suggestion
```

File: source/_integrations/pegel_online.markdown [hunk 4]
```diff
@@ -18,11 +18,39 @@ ha_integration_type: service
...
+Longitude:
+    description: "The longitude of the center of the search area (_automatically filled by the location picker_)."
+    required: true
```
Comment (frenck) [inline source/_integrations/pegel_online.markdown]:
```suggestion
```

File: source/_integrations/pegel_online.markdown (hunk 2 above)
Comment (frenck) [inline source/_integrations/pegel_online.markdown]:
```suggestion
```

File: source/_integrations/pegel_online.markdown [hunk 5]
```diff
@@ -18,11 +18,39 @@ ha_integration_type: service
...
+    description: "Search radius (_in km_)"
+    required: true
+{% endconfiguration_basic %}
+
+### Step 2 - Station selection
+
+{% configuration_basic %}
+Station:
+    description: "Select the measurement station you want to add."
+    required: true
```
Comment (frenck) [inline source/_integrations/pegel_online.markdown]:
```suggestion
```

File: source/_integrations/pegel_online.markdown [hunk 6]
```diff
@@ -34,3 +62,31 @@ Currently, the integration supports the following sensors:
...
+actions:
+  - action: notify.persistent_notification
+    metadata: {}
+    data:
+      message: Flood alert level 2 reached!
+```
+
+## Troubleshooting
+
+In any case, when reporting an issue, please enable [debug logging](/docs/configuration/troubleshooting/#debug-logs-and-diagnostics), restart the integration, and as soon as the issue re-occurs stop the debug logging again (_download of debug log file will start automatically_). Further _if still possible_, please also download the [diagnostics](/integrations/diagnostics) data. If you have collected the debug log and the diagnostics data, provide them with the issue report.
```
Comment (frenck) [inline source/_integrations/pegel_online.markdown]:
```suggestion
In any case, when reporting an issue, please enable [debug logging](/docs/configuration/troubleshooting/#debug-logs-and-diagnostics), restart the integration, and as soon as the issue reoccurs stop the debug logging again (_download of debug log file will start automatically_). Further _if still possible_, please also download the [diagnostics](/integrations/diagnostics) data. If you have collected the debug log and the diagnostics data, provide them with the issue report.
```

**Review by frenck — APPROVED** [review]
Thanks, @mib1185 👍 

../Frenck

**Pull request merged**